docs/zh/tutorial/images/openmldb_sql_tutorial_1.drawio

```diff
@@ -0,0 +1 @@
HMf4WxkM8+W+apS9JMErjNCu2zPhQ/EerdNyd7Lv9CDMSvtZgYrv3Y5jOQ5KtaRX2ahkZLFJsna2vttzZJiub1Zgzbft+YDHkGe7TqvVtl9IF1qs9etiEPXywy/7IER/O0iz6mXdUzPqk3o3F+iqaxzihsYODvaJhWuSKvIikC7YUh8+ELX5LCUnnbCVju0FvJCHI0sWfOJuGZZXnKI7LLk3SJAdskUYJKfaaNaR/dP+O9HtLs+g3G9F1tF2nf3n1jIzSZEkyCmHebIiXZBUuyVnZMTqzw2AxurKiX4gTC3BCq7eRQr8/iXD8B03YOJkWcT4j85jF5GoWkfBpgSd51RU9rGyiPk/veNu3DV3ZD5eU7t3nuMjIsygIwuSsPWh27sFalzk9e4w1tt2PvVvDMU17CSY06uj3XQIMqu08ngwbkPGi0DjqwCA6KQMPoDJXqPRCBZmSsOIAVOgROVl+JdE8VMT0IKY606r1MRqcFZlOzfHIL3CQsoEGz4lipgczRsNw0z0rMl1a40HMoC3NrBcqzfRBxmxIM0Iig+AgZpGlP6JkooDpBYwnCzDwBGkSkbWCpQcsliynSC5gpZUTpeOup+MacOSr4+BBqF21S5BQ3M4dKHoCKYdhNTL+WoaZr+hQQg7mPQjLo8ojysc15RUEUBnog8Gdnv/Rcpp3DF9n/1MEKT/XZFvgmRDS7xUuSs01wQLnnoz8p08KFiXlGmCBAxmFitJxjQMZ+Mvi04xu7gxHChil5BqAgb8RtYKinNz1nBxqONjxlXIIDlkMmXNKFTlKy6GmebZ2nIfMcoETujzNl5mq27xAP6j+miJJKbyGNAm9zGP78UmhIrXDg/OjmpJQ5fWM3OuhutdTielk2mTyffBUayC57+vPizzCDw6Rfv/tScGihF8TLHDYI7fwO4IVaYxf06QqOOzZWkA1yjkZLmnsoKFk4E3KQOvaMtCAMtCUOYcY3XOI8DlDyUBOJMkmA6HgeZT6F4j+qEgjAw04GeugDHSUDDw/bRLJQAPKQGTJLQP78yKNDDTgObvkMrA/LNLIQANeDSW3DDyCFVlkYPmRSgbyg0saGWi+cb9DJQOvIwMbkhtfGWiWMbPNOja3HFLs4gfPtnklkmmGg4j24R5M1Q1SC6D/C8lkxr5Tx44vw0spxDuz4c6st6oQufN3YbYkk4omtNUcLxIWFB5pNKPZc85hoRmN29GMgvInkXg0oejmOgtRUIKkUZEmFNdcL0QWlB9p7KQJVfbwV0XPifRI4yuh2x6G0a9RMlUInYaQNFbSah/rKCt5PSt5A8/0gJPoXZVVmkJHGcc7CypsZRw5sSWZcbSg3OZ4TbOg8EhjHK1us6sr4+hqtzWxUVD+JDKOFnTeljKOJxMkjXG0uk3N3lhIlawugJo0ctKCclvJyZPpkUVOWg1qW8nJcyAkj5z0WlFRcvKKcvLqTzix4UTtdlTkzCpe514VPYvY0GQrOcmJLcnkpA09OMdrrAWFRxo5aXebt70rJwdKTl6YP4nkpA31ONKVnTwZIWnspA3tNs9LswXFRxrjaEO5rYzjyfTIYhxt6Kubxk+VhVSjpQvQJo2ctNunwyk5eT05aVz9SS82nJU94JlVnLFvj/tDwzGr2OqRzOWuKDejg5wc3sARSyi2JJOTTsPTYLjKSRHhkUZOOlBtH5STSGMzJ42bkZMi8ieRnHSgHuf6hGhBCZLGTTrXfYaMoPhI4yYdqLb5XugvJD2yuEkHmm3uT5YWlCFpjKPjtKKijOMVjePVHyfjwKnWlsoqTaGjjOOdA/W0Mo6c2JLMOJZjr6vdHVJEeKQxjm63edu7d4fUb2o6pIj8SWQcXei8ed8dUkSCpDGOLjTWyjiejI80xtGFwloZx5PpkcU4ut1urn0rD7MRFDdp5KTb/swSJSevKCe7Pt6mfOTn2eWkC2dlt3vsC2QVz3nQadzecFYpQ+d0OXmoE/tnkf3WeGQRaLJvWU4KxVY/Ofn+UYMenO+NJEWEp6ucfPf0lIfVXnLytq7VFpG/znLyeAA7NceDQKjHDc5yUkSCusrJ4wHq0hoPfqDc5iwnRcSnq5x8//hAt81XTgpJT0c5+f7p6XYf7huSkyLi1lVOvv/RurpW+xblpFk+6PpqMyc9KAGQGkI3xo6aOnmHdHjWn6tIrg9GEooYySZEIr3hGmyuU7VFpEeaGZFIh+fs7+3xNSICKNGUSKQ3XYWtADoNIGlmRCL96hdhi8iPNFMikX7tq7CFxEeWOZFI7/b0LOUdL8qbNJMivaY5uHvshEngZ1nRU5MYL5fRZJeaoq/C0jeGrxH5Ulv+J1++t9jaw2vtpYdSSNLuy9Zf6iu1d+Wr27cVa9X7gmn4xDY0zcgsnaYJjsfb0p5icJm+ZJPwUD22d0hpP9sqMuLyDTzIW40Iq4GIsiwLY0yiH/W2DkL3e25ftzhX1zkyrYncPdI2X5y9a0sZbMjeawjtNbTZMaAhyg9e16oxO9y+wcbe57Bwad2uvfreTnW6sNmA84ZOQ6Ye25qva0MnX/BGmj/ytLGjeb7mmisUfEV6kK/7vubRKpbmftB8o6hraUM9X3AfNH8MI5Bi9DnPjbuBh+NomuRRSeEOaaIc5gkumuDYZy/MaWItwiALl9HPWgrftfMPPVJpmS/2U+lA3z1Q0DL9YIrV7+mZmrvby+eBvXzSUvmW9Pl5GZI9IM6DQMP9nA8jcFz3qwS8f7vnNxOwiW4rA1dmvpz27hybgq29hgYXSsF7qX4vp75dH3HIwUjvEoF0S6oQ1L8iFYOnxWA5nH57EGTfVAxWkVL9uHtkCFax2xbLFwpB0+oXgiafEGy6IuK06LlYJBwTl2eOnvIKvTejx76Rc4gS8mPPGfagtPQLBYvZHJStwbJf3+ASLA33Njp4vBp83ay943OGbYY4w0lDdeOwdTmwOs/4zEA7zSJzt4VjziHoapampF49w4vZYxrknTL+Hw==</diagram></mxfile>
```

Comment:
Are the drawio files in need of docs?

Reply:
> Are the drawio files in need of docs?

it should be okay we put the source of images here? the compiled docs will not show any about the draw.io file.